gplugins/lumerical/tests/test_lumerical_utils.py

```diff
@@ -0,0 +1,249 @@
+                sidewall_angle_tolerance=None,
+                width_to_z=0.0,
+                z_to_bias=None,
+                mesh_order=9,
+                layer_type="background",
+                mode=None,
+                into=None,
+                resistivity=None,
+                bias=None,
+                derived_layer=None,
+                info={},
+                background_doping_concentration=None,
+                background_doping_ion=None,
+                orientation="100",
+            ),
+            "core": LayerLevel(
+                name=None,
+                layer=(1, 0),
+                thickness=0.22,
+                thickness_tolerance=None,
+                zmin=0.0,
+                zmin_tolerance=None,
+                material="si",
+                sidewall_angle=10.0,
+                sidewall_angle_tolerance=None,
+                width_to_z=0.5,
+                z_to_bias=None,
+                mesh_order=2,
+                layer_type="grow",
+                mode=None,
+                into=None,
+                resistivity=None,
+                bias=None,
+                derived_layer=None,
+                info={"active": True},
+                background_doping_concentration=100000000000000.0,
+                background_doping_ion="Boron",
+                orientation="100",
+            ),
+            "substrate": LayerLevel(
+                name=None,
+                layer=(99999, 0),
+                thickness=10.0,
+                thickness_tolerance=None,
+                zmin=-13.0,
+                zmin_tolerance=None,
+                material="si",
+                sidewall_angle=0.0,
+                sidewall_angle_tolerance=None,
+                width_to_z=0.0,
+                z_to_bias=None,
+                mesh_order=101,
+                layer_type="background",
+                mode=None,
+                into=None,
+                resistivity=None,
+                bias=None,
+                derived_layer=None,
+                info={},
+                background_doping_concentration=100000000000000.0,
+                background_doping_ion="p",
+                orientation="100",
+            ),
+            "n++": LayerLevel(
+                name="n++",
+                layer=(5, 0),
+                thickness=0.22,
+                zmin=0,
+                layer_type="doping",
+                background_doping_concentration=1e17,
+                background_doping_ion="n",
+            ),
+        }
+    )
+
+    # Create LBR process file
+    process_file_lumerical2021 = to_lbr(layer_map, layerstack=layerstack_lumerical2021)
+
+    # Check process file in Lumerical MODE
+    try:
+        import lumapi
+    except Exception as err:
+        assert (
+            False
+        ), f"{err}\nUnable to import lumapi. Check sys.path for location to lumapi.py."
+
+    success = False
+    message = ""
+    try:
+        mode = lumapi.MODE(hide=False)
```

Comment:
I think hide for these should be set by a global/runtime `DEBUG` flag, else this just clutters the desktop for big sweeps. I was using a `DEBUG_LUMERICAL` var, you're welcome to refactor but I think we want to avoid spawning tons of windows that likely won't be looked at with regular usage